x-pack/plugins/secops/public/components/empty_value/empty_value.test.tsx

```diff
@@ -4,56 +4,86 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import euiDarkVars from '@elastic/eui/dist/eui_theme_dark.json';
 import { mount } from 'enzyme';
 import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import { mountWithIntl } from 'test_utils/enzyme_helpers';
+
 import {
-  defaultToEmpty,
   defaultToEmptyTag,
+  getEmptyString,
+  getEmptyStringTag,
   getEmptyTagValue,
   getEmptyValue,
-  getOrEmpty,
   getOrEmptyTag,
 } from '.';
 
 describe('EmptyValue', () => {
+  const theme = () => ({ eui: euiDarkVars, darkMode: true });
+
   describe('#getEmptyValue', () =>
     test('should return an empty value', () => expect(getEmptyValue()).toBe('--')));
 
+  describe('#getEmptyString', () => {
+    test('should turn into an empty string place holder', () => {
+      const wrapper = mountWithIntl(
+        <ThemeProvider theme={theme}>
+          <p>{getEmptyString()}</p>
+        </ThemeProvider>
+      );
+      expect(wrapper.text()).toBe('(Empty String)');
+    });
+  });
+
   describe('#getEmptyTagValue', () => {
     const wrapper = mount(<p>{getEmptyTagValue()}</p>);
     test('should return an empty tag value', () => expect(wrapper.text()).toBe('--'));
   });
 
-  describe('#getOrEmpty', () => {
-    test('should default empty value when a deep rooted value is null', () => {
+  describe('#getEmptyStringTag', () => {
+    test('should turn into an span that has length of 1', () => {
+      const wrapper = mountWithIntl(
+        <ThemeProvider theme={theme}>
+          <p>{getEmptyStringTag()}</p>
+        </ThemeProvider>
+      );
+      expect(wrapper.find('span')).toHaveLength(1);
+    });
+
+    test('should turn into an empty string tag place holder', () => {
+      const wrapper = mountWithIntl(
+        <ThemeProvider theme={theme}>
+          <p>{getEmptyStringTag()}</p>
+        </ThemeProvider>
+      );
+      expect(wrapper.text()).toBe(getEmptyString());
+    });
+  });
+
+  describe('#defaultToEmptyTag', () => {
+    test('should default to an empty value when a deep rooted value is null', () => {
       const test = {
         a: {
           b: {
             c: null,
           },
         },
       };
-      expect(getOrEmpty('a.b.c', test)).toBe(getEmptyValue());
+      const wrapper = mount(<p>{defaultToEmptyTag(test.a.b.c)}</p>);
```

Comment:
What is the intent of using the nested object `test` in the above and below tests if you just end up passing the specific `null`/`undefined` value directly to `defaultToEmptyTag()`? This would be more suitable for testing `getOrEmptyTag()` as it is the only utility that uses a lodash `get`?

Reply:
Agreed, fixing it to just pass null and undefined below